src/uct/ib/rc/base/rc_iface.c

```diff
@@ -585,7 +585,7 @@ UCS_CLASS_INIT_FUNC(uct_rc_iface_t, uct_rc_iface_ops_t *ops, uct_md_h md,
                                 &uct_rc_fc_pending_mpool_ops,
                                 "pending-fc-grants-only");
         if (status != UCS_OK) {
-            goto err_destroy_stats;
+            goto err_destroy_srq;
```

Comment:
are tm.stats released in case of error here?

Reply:
tm.stats allocated in rc_mlx5_common constructor, later...

Reply:
i see it's called from uct_rc_mlx5_init_rx_tm() after creating srq